src/audits/offline/service-worker.js

```diff
@@ -47,12 +47,7 @@ class ServiceWorker extends Audit {
    */
   static audit(artifacts) {
     const registrations = artifacts.serviceWorkers.versions;
-    const activatedRegistrations = registrations.filter(reg => {
-      return reg.status === 'activated' &&
-          reg.scriptURL.startsWith(artifacts.url);
-    });
-
-    return ServiceWorker.generateAuditResult(activatedRegistrations.length > 0);
+    return ServiceWorker.generateAuditResult(registrations.length > 0);
```

Comment:
We don't need the old check; we do all the filtering in the gather for active registrations that match the current origin.